gno.land/pkg/gnoweb/webclient_html.go

```diff
@@ -238,23 +239,57 @@ func (s *HTMLWebClient) FormatSource(w io.Writer, fileName string, src []byte) e
 	case ".mod":
 		lexer = lexers.Get("gomod")
 	default:
-		lexer = lexers.Get("txt") // Unsupported file type, default to plain text.
+		lexer = lexers.Get("txt") // Unsupoorted file type, default to plain text.
 	}
 
 	if lexer == nil {
 		return fmt.Errorf("unsupported lexer for file %q", fileName)
 	}
 
+	// Format with Chroma
+	var buf bytes.Buffer
 	iterator, err := lexer.Tokenise(nil, string(src))
 	if err != nil {
 		return fmt.Errorf("unable to tokenise %q: %w", fileName, err)
 	}
 
-	if err := s.Formatter.Format(w, s.chromaStyle, iterator); err != nil {
+	if err := s.Formatter.Format(&buf, s.chromaStyle, iterator); err != nil {
 		return fmt.Errorf("unable to format source file %q: %w", fileName, err)
 	}
 
-	return nil
+	formatted := buf.String()
+
+	// Process .gno files to add links
+	if strings.HasSuffix(fileName, ".gno") {
+		// Find and process string spans containing gno.land imports
+		lines := strings.Split(formatted, "\n")
+		for i, line := range lines {
+			if strings.Contains(line, "gno.land/") {
+				// Look for chroma string spans
+				start := strings.Index(line, `<span class="chroma-s">&#34;gno.land/`)
+				if start != -1 {
+					prefix := line[:start]
+					rest := line[start:]
+					end := strings.Index(rest, `&#34;</span>`)
+					if end != -1 {
+						// Extract the path of gnoland import
+						importPath := rest[len(`<span class="chroma-s">&#34;`):end]
+						urlPath := strings.TrimPrefix(importPath, "gno.land/")
+
+						// Create new span with link
+						replacement := fmt.Sprintf(`<span class="chroma-s"><a href="/%s$source" class="text-blue-600 hover:underline">&#34;%s&#34;</a></span>`,
```

Comment:
```suggestion
						replacement := fmt.Sprintf(`<span class="chroma-s"><a href="/%s$source" class="hover:underline">&#34;%s&#34;</a></span>`,
```

Reply:
Actually, you shouldn't rely on Tailwind colors here since the styling of "source code" is managed by Chroma. Also, since `text-blue-600` does not exist in our theme and because `color: inherit;` is the default behavior for `a` element, defining a specific color might not be necessary.